ansible_mitogen/transport_config.py

```diff
@@ -686,6 +686,7 @@ def become_pass(self):
     def password(self):
         return optional_secret(
             self._host_vars.get('ansible_ssh_pass') or
+            self._host_vars.get('ansible_ssh_password') or
```

Comment:
Based on some adhoc testing this will not match the behaviour of vanilla Ansible. `ansible_ssh_password` should have the highest priority, then `ansible_ssh_pass`, then `ansible_password`. This playbook is how I confirmed
```
- hosts: localhost
  gather_facts: false
  vars:
    ansible_user: mitogen__user1
    ansible_port: 32768
    ansible_python_interpreter: python3

    ansible_password: wrong
    ansible_ssh_pass: wrong
    ansible_ssh_password: user1_password

  tasks:
    - meta: reset_connection
    - ping:
    - debug:
        var: ansible_version.full
```
```
$ ansible-playbook -i localhost, issue1106.yml
PLAY [localhost] *************************************************************

TASK [meta] ******************************************************************

TASK [ping] ******************************************************************
ok: [localhost]

TASK [debug] *****************************************************************
ok: [localhost] => {
    "ansible_version.full": "2.17.2"
}

PLAY RECAP *******************************************************************
localhost                  : ok=2    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
```